src/redux/action-creator/adminList.ts

```diff
@@ -51,35 +51,58 @@ export const fetchAdminList = () => {
     };
 };
 
-export const deleteAdminAction = (adminAddress: string) => {
+export const deleteAdminAction = (adminAddress: string, stargateClient: SigningStargateClient) => {
     return async (dispatch: Dispatch<AdminActions>): Promise<void> => {
         try {
             const kepler = await getKeplr();
```

Comment:
Keplr is in wallet state to make sure the user has connected the wallet

Reply:
refactored